bin/install/install-dcos-cli.sh

```diff
@@ -15,6 +15,10 @@ fi
 ARGS=( "$@" );
 
 VIRTUAL_ENV_PATH=$(python -c "import os; print(os.path.realpath('"${ARGS[0]}"'))")
+if [[ $VIRTUAL_ENV_PATH =~ \  ]];
+	then echo "Installation cancelled - spaces are not allowed in the DCOS-CLI installation path. Please install the DCOS-CLI in a path that does not contain spaces.";
```

Comment:
Hey @ytalashkocv - would you mind updating the error message to:
"Spaces are not permitted in the installation path. Please try again with another path."